npc/re/quests/quests_aldebaran.txt

```diff
@@ -45,41 +46,18 @@ c_tower1,232,222,5	script	Belljamin Button#ct	4_M_RUSMAN2,{
 	next;
 	switch (select("Don't ask", "Go to 2nd floor of warped tower", "Go to 3rd floor of warped tower")) {
 	case 1:
-		mes("[Belljamin Button]");
-		mes("Phew...You've made good choice. The place is so brutal so you should probably get back to a safe area.");
-		close;
-	case 2:
-		mes("[Belljamin Button]");
-		mes("Be careful with your choice.");
-		next;
-		mes("[Belljamin Button]");
-		mes("I need ^0000FF10,000z^000000 to let you in. It's a pretty dangerous place to be.");
-		next;
-		switch (select("Quit", "Pay 10,000z", "Why do you need money?")) {
-		case 1:
-			mes("[Belljamin Button]");
-			mes("Phew...You've made good choice. The place is so brutal so you should probably get back to a safe area.");
-			close;
-		case 2:
-			if (Zeny < 10000) {
-				mes("[Belljamin Button]");
-				mes("Really? You don't have money and asked me to take you there?");
-				close;
-			}
-			Zeny -= 10000;
-			mes("[Belljamin Button]");
-			mes("I'm not sure how you'll keep track of time in there but be as careful as you can.");
-			close2;
-			warp "c_tower2_", 268, 26;
-			end;
-		case 3:
-			mes("[Belljamin Button]");
-			mes("I am here all day for the safety of adventurers. I always alert them about the danger. That's why I need some pay.");
-			emotion e_swt;
-			close;
-		}
 		break;
+	case 2:
 	case 3:
+		if (@menu == 2) {
```

Comment:
Okay, this horrible feature which added to switch code (save to character temporary variable).
I would prefer if you assign the switch return to a NPC temporary variable .@